test/performance/specs/post-editor.spec.js

```diff
@@ -668,7 +668,10 @@ test.describe( 'Post Editor Performance', () => {
 
 				const startTime = performance.now();
 
-				await page.getByRole( 'tab', { name: 'Test' } ).click();
+				const oldLocator = page.getByRole( 'button', { name: 'Test' } );
+				const newLocator = page.getByRole( 'tab', { name: 'Test' } );
+
+				await oldLocator.or( newLocator ).click();
```

Comment:
Let's add an inline comment. It helps with confidence when we might want to remove the old locator.